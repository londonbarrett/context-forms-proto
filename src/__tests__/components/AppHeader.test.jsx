import React from 'react';
import { mount, render, shallow } from 'enzyme';
import AppHeader from '../../components/AppHeader';

it('Renders correctly', () => {
  const wrapper = shallow(<AppHeader />);
  expect(wrapper).toMatchSnapshot();
});

it('Renders header title', () => {
  const wrapper = mount(<AppHeader />);
  const title = wrapper.find('h1').text();
  expect(title).toEqual('React Context Forms');
});

it('Renders React logo', () => {
  const wrapper = render(<AppHeader />);
  const logo = wrapper.html();
  expect(logo).toMatch(/logo.svg/);
});

test('Logo has alt attribute', () => {
  const wrapper = render(<AppHeader />);
  const alt = wrapper.find('img').prop('alt');
  expect(alt.length).toBeGreaterThan(0);
});
