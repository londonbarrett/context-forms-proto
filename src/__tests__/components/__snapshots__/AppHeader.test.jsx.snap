// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<styled.header>
  <styled.img
    alt="logo"
    src="logo.svg"
  />
  <h1>
    React Context Forms
  </h1>
</styled.header>
`;
